swagger/license/directoryPackageBundle.json

```diff
@@ -0,0 +1,281 @@
+                      "status" : {
+                        "type" : "boolean",
+                        "example" : true
+                      },
+                      "data" : {
+                        "type" : "object",
+                        "properties" : {
+                          "purchasables" : {
+                            "type" : "array",
+                            "items" : {
+                              "$ref" : "https://vcita.github.io/developers-hub/entities/license/directoryPackageBundle.json"
+                            }
+                          }
+                        }
+                      },
+                      "page" : {
+                        "type" : "integer"
+                      },
+                      "per_page" : {
+                        "type" : "integer"
+                      },
+                      "total" : {
+                        "type" : "integer"
+                      },
+                      "total_pages" : {
+                        "type" : "integer"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "security" : [ {
+            "api_key" : [ ]
+          } ],
+          "x-auth-type" : [ "Admin", "Directory", "Business", "app" ],
+          "tags" : [ "directoryPackage" ],
+          "description" : "List all directory package bundles in scope.\n\nAvailable for **Directory, business, and app Tokens** \n"
+        },
+        "post" : {
+          "operationId" : "create",
+          "requestBody" : {
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "https://vcita.github.io/developers-hub/entities/license/directoryPackageBundle.json"
+                }
+              }
+            },
+            "required" : true
+          },
+          "responses" : {
+            "201" : {
+              "description" : "",
+              "content" : {
+                "application/json" : {
+                  "schema" : {
+                    "type" : "object",
+                    "properties" : {
+                      "data" : {
+                        "$ref" : "https://vcita.github.io/developers-hub/entities/license/directoryPackageBundle.json"
+                      },
+                      "status" : {
+                        "type" : "string",
+                        "example" : "directory package Bundle created"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "summary" : "Create a directory package Bundle",
+          "description" : "Create a new directory package Bundle for a sepcific directory package.\n\n**Available only for Admin token**\n",
+          "security" : [ {
+            "api_key" : [ ]
+          } ],
+          "x-auth-type" : [  "Admin" ],
+          "tags" : [ "directoryPackage" ]
```

Comment:
Tags are the way readme groups the items. Use friendly names "Directory Package"